src/Navbar.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable arrow-body-style */
```

Comment:
- [ ] `required`
- [ ] Customizing the Linters by `Disabling the eslint` defeats the purpose of using it please enable it.

Reply:
@mwanawabangona same for here like above but also i disabled arrow-body-style */ because the function is perfect but they want me to write the arrow function in another which I don't really know how to do and I don't think is important.